lib/routes/data/styles.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+/*
+ * Sample Data for Styles Route
+ *
+ * @returns {object} - Object containing sample data to use when rendering samples in the Style Guide
+ */
+
+module.exports = {
+  myLatest: {
```

Comment:
As we're now just starting to build out a proper style guide, I would like our data to follow the following semantics:

```javascript
componentData: [ // name of the component, plus Data
  {
    name: 'Sample component instance name', // when we display multiple versions of a component, we need subtitles for the components
    data: {}, // The actual data for use in this component
  },
];
```

Then, in the Style Guide generation, we'll do something like the following:

```html
<h2 class="_style-guide--title">Component</h2>
  {% from "components/component.html" import component %}
  {% for example in componentData %}
  <h3 class="_style-guide--instance>{{example.name}}</h3>
  {{component(example.data)}}
</section>